samples/wifi/shell/README.rst

```diff
@@ -156,6 +156,34 @@ Supported CLI commands
      - | Configure Wi-Fi power save for wakeup mode
        | dtim - Wakeup mode for the DTIM interval
        | listen_interval - Wakeup mode for the Listen interval
+   * - mode
+     - | This command may be used to set the Wi-Fi device into a specific mode of operation
+       | parameters:
+       | [-i] : Interface index - optional argument
```

Comment:
This needs to match the actual output, so, I guess we should fix it in upstream and pull.

Reply:
will raise it a review for it upstream. But it is not a pull to this readme.rst.

Reply:
Then I would suggest reverting this to keep doc and code same, and whenever you pull you can update docs.

Reply:
:( - okay ...will change it back

Reply:
as per discussion, reverted back to original format till upstream is not updated.